spec/controllers/account/users_controller_spec.rb

```diff
@@ -1,4 +1,229 @@
 require 'rails_helper'
 
 RSpec.describe Account::UsersController, type: :controller do
+  render_views
+
+  let!(:admin) { create(:user, :super_admin, first_name: 'Kelly', role: User::SUPER_ADMIN, organization: user.organization) }
```

Comment:
confusing,

let(:admin) BUT returns super_admin

rename let to super_admin to be consistent

Reply:
and why super_admin has organization?